bin.src/night_report.py

```diff
@@ -0,0 +1,132 @@
+
+    unislicer = slicers.UniSlicer()
+    for sql in sqls:
+
+        # Number of exposures
+        metric = metrics.CountMetric('expMJD', metricName='N visits')
+        bundle = metricBundles.MetricBundle(metric, reg_slicer, sql)
+        bundleList.append(bundle)
+        metric = metrics.CountMetric('expMJD', metricName='N visits alt az')
+        bundle = metricBundles.MetricBundle(metric, altaz_slicer, sql, plotFuncs=plotFuncs_lam)
+        bundleList.append(bundle)
+
+        metric = metrics.MeanMetric('expMJD', metricName='Mean Visit Time')
+        bundle = metricBundles.MetricBundle(metric, reg_slicer, sql)
+        bundleList.append(bundle)
+        metric = metrics.MeanMetric('expMJD', metricName='Mean Visit Time alt az')
+        bundle = metricBundles.MetricBundle(metric, altaz_slicer, sql, plotFuncs=plotFuncs_lam)
+        bundleList.append(bundle)
+
+        metric = metrics.CountMetric('expMJD', metricName='N_visits')
+        bundle = metricBundles.MetricBundle(metric, unislicer, sql)
+        bundleList.append(bundle)
+
+        # Need pairs in window to get a map of how well it gathered SS pairs.
+
+    # Moon phase.
+
+    metric = metrics.NChangesMetric(col='filter', metricName='Filter Changes')
+    bundle = metricBundles.MetricBundle(metric, unislicer, 'night=%i' % night)
+    bundleList.append(bundle)
+
+    metric = metrics.OpenShutterFractionMetric()
+    bundle = metricBundles.MetricBundle(metric, unislicer, 'night=%i' % night)
+    bundleList.append(bundle)
+
+    metric = metrics.MeanMetric('slewTime')
+    bundle = metricBundles.MetricBundle(metric, unislicer, 'night=%i' % night)
+    bundleList.append(bundle)
+
+    metric = metrics.MinMetric('slewTime')
+    bundle = metricBundles.MetricBundle(metric, unislicer, 'night=%i' % night)
+    bundleList.append(bundle)
+
+    metric = metrics.MaxMetric('slewTime')
+    bundle = metricBundles.MetricBundle(metric, unislicer, 'night=%i' % night)
+    bundleList.append(bundle)
+
+    # Make plots of the solar system pairs that were taken in the night
+    metric = metrics.PairMetric()
+    sql = 'night=%i and (filter ="r" or filter="g" or filter="i")' % night
+    bundle = metricBundles.MetricBundle(metric, reg_slicer, sql)
+    bundleList.append(bundle)
+
+    metric = metrics.PairMetric(metricName='z Pairs')
+    sql = 'night=%i and filter="z"' % night
+    bundle = metricBundles.MetricBundle(metric, reg_slicer, sql)
+    bundleList.append(bundle)
+
+    # Plot up each visit
+    metric = metrics.NightPointingMetric()
+    slicer = slicers.UniSlicer()
+    sql = sql = 'night=%i' % night
+    plotFuncs = [plots.NightPointingPlotter()]
+    bundle = metricBundles.MetricBundle(metric, slicer, sql, plotFuncs=plotFuncs)
+    bundleList.append(bundle)
+
+    return metricBundles.makeBundlesDictFromList(bundleList)
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description='Python script to run the science performance metrics.')
```

Comment:
Can you fix this description? Probably not the same as the science performance metrics that opsim is used to.